lib/elixir/lib/gen_server.ex

```diff
@@ -758,6 +758,38 @@ defmodule GenServer do
               | {:error, reason :: term}
             when old_vsn: term | {:down, term}
 
+  @doc """
+  This function is called by a `GenServer` process in the following situations:
```

Comment:
I copied the [erlang docs](https://www.erlang.org/doc/man/gen_server#Module:format_status-1) and tweaked them for Elixir